### PR TITLE
Fix percent display in markdown and prettify the output of absolute values

### LINKED ISSUE #1: Markdown output does not print percentages correctly
A factor of 100 is missing when displaying the relative change in `repometrics::output::markdown`.

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
Close #1

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -9,3 +9,37 @@ pub fn print_comparisons(format: OutputFormat, comparisons: &Comparisons) {
...
+    let printed = i.to_string();
+
+    let tmp: Vec<char> = printed.chars().collect();
+    tmp.into_iter()
+        .rev()
+        .enumerate()
+        .rev()
+        .flat_map(|(idx, c)| {
+            if idx % 3 == 0 && c != '-' && idx != 0 {
+                [c].into_iter().chain(Some('_'))
```
Comment (robin-nitrokey) [inline src/output.rs]: I think a space or comma would be easier to read, especially in a sans-serif font.  What do you think?

Comment (sosthene-nitrokey) [reply src/output.rs]: Let's look at the variations:

# Current

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1199399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2001105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584604 | :white_check_mark: | -3600 (-0.01%) |

# underscore

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1_199_399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2_001_105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431_492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584_604 | :white_check_mark: | -3600 (-0.01%) |

# underscore + monospace

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | `1_199_399` | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | `2_001_105` | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | `431_492` | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | `584_604` | :white_check_mark: | -3600 (-0.01%) |

# comma + monospace

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | `1,199,399` | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | `2,001,105` | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | `431,492` | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | `584,604` | :white_check_mark: | -3600 (-0.01%) |

# comma

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1,199,399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2,001,105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431,492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584,604 | :white_check_mark: | -3600 (-0.01%) |

Comment (sosthene-nitrokey) [reply src/output.rs]: Ok comma is indeed better. The numbers themselves appear to be monospace so the width of the number is proportional to the number of characters as expected.

Comment (robin-nitrokey) [reply src/output.rs]: I prefer comma.

Comment (sosthene-nitrokey) [reply src/output.rs]: I changed to commas.

**Pull request merged**